src/player.js

```diff
@@ -8,11 +8,24 @@ class Player {
             this._points = 30
         } else {
             this._points = 15
-        } 
+        }
     }
 }
 
+// Easier way to understand the tests
+
+const points = (a) => {
```

Comment:
please rename this variable

Reply:
Changed it! Under Commit commit bb8045e0321f5c9f009e077829602ff25a9d59d5
  8